cli/src/rpc.rs

```diff
@@ -94,7 +94,7 @@ pub struct FullDeps<C, P, SC, B> {
 pub type IoHandler = jsonrpc_core::IoHandler<sc_rpc::Metadata>;
 
 /// Instantiate all Full RPC extensions.
-pub fn create_full<C, P, SC, B>(deps: FullDeps<C, P, SC, B>) -> jsonrpc_core::IoHandler<sc_rpc_api::Metadata>
+pub fn create_full<C, P, SC, B>(deps: FullDeps<C, P, SC, B>) -> IoHandler
```

Comment:
not sure, what this is for... rest lgtm

Reply:
little drive by fix: I noticed the type was already aliased as `pub type IoHandler = jsonrpc_core::IoHandler<sc_rpc::Metadata>;` so used it